### DISCUSSION TITLE
Failed to import the site module in Windows Environment

### DISCUSSION THREAD

**Discussion by BradenM** [ymyzk/tox-gh-actions#164] [Q&A] [answered]
Hi,

Not entirely sure if this is directly related to `tox-gh-actions` or not, but I seem to be having an issue with `tox` in Windows environments only.

In both `macOS-10.14` and `ubuntu-18.04` my action passes as expected, but in `windows-2016` I am met with: 

```
 Fatal Python error: initsite: Failed to import the site module
Traceback (most recent call last):
  File "D:\a\micropy-cli\micropy-cli\.tox\py37\lib\site.py", line 769, in <module>
    main()
  File "D:\a\micropy-cli\micropy-cli\.tox\py37\lib\site.py", line 746, in main
    paths_in_sys = addsitepackages(paths_in_sys)
  File "D:\a\micropy-cli\micropy-cli\.tox\py37\lib\site.py", line 279, in addsitepackages
    addsitedir(sitedir, known_paths)
  File "D:\a\micropy-cli\micropy-cli\.tox\py37\lib\site.py", line 202, in addsitedir
    addpackage(sitedir, name, known_paths)
  File "D:\a\micropy-cli\micropy-cli\.tox\py37\lib\site.py", line 170, in addpackage
    exec(line)
  File "<string>", line 1, in <module>
  File "D:\a\micropy-cli\micropy-cli\.tox\py37\lib\site-packages\win32\lib\pywin32_bootstrap.py", line 14, in <module>
    for maybe in site.getsitepackages():
AttributeError: module 'site' has no attribute 'getsitepackages'
```

Followed by:
`ERROR:   py37: InvocationError for command 'D:\a\micropy-cli\micropy-cli\.tox\py37\Scripts\python.EXE' -m pip freeze (exited with code 1)`

The full checksuite can be viewed [here](https://github.com/BradenM/micropy-cli/pull/69/checks?check_run_id=296583737)
The full workflow can be found [here](https://github.com/BradenM/micropy-cli/blob/master/.github/workflows/main.yml)
`tox.ini` file can be found [here](https://github.com/BradenM/micropy-cli/blob/master/tox.ini).

For now I will be trying a different windows environment to test if the issue persists. Thanks.

**Comment by BradenM** [comment]
Quick update, issue also seems to be present in `windows-latest` environment.
Checksuite can be viewed [here](https://github.com/BradenM/micropy-cli/runs/296602366)

**Comment by BradenM** [comment, marked as answer]
Not exactly sure what the original issue or origin was, but it does seem to be fixed now.

**Comment by ymyzk** [comment]
@BradenM sorry for slow response. Feel free to file another ticket or reopen this if you need help 😃
